client/src/templates/Challenges/quiz/exit-quiz-modal.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { createSelector } from 'reselect';
+import { useTranslation } from 'react-i18next';
+import { Button, Modal, Spacer } from '@freecodecamp/ui';
+
+import { closeModal } from '../redux/actions';
+import { isExitQuizModalOpenSelector } from '../redux/selectors';
+
+interface ExitQuizModalProps {
+  closeExitQuizModal: () => void;
+  isExitQuizModalOpen: boolean;
+  onExit: () => void;
+}
+
+const mapStateToProps = createSelector(
+  isExitQuizModalOpenSelector,
+  (isExitQuizModalOpen: boolean) => ({
+    isExitQuizModalOpen
+  })
+);
+
+const mapDispatchToProps = (dispatch: Dispatch) =>
+  bindActionCreators(
+    {
+      closeExitQuizModal: () => closeModal('exitQuiz')
+    },
+    dispatch
+  );
```

Comment:
```suggestion
const mapDispatchToProps = {
  closeExitQuizModal: () => closeModal('exitQuiz')
};
```
Same thing, but [less cruft](https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object).